front-end/src/components/Admin/Grid/Filter/index.js

```diff
@@ -5,68 +5,72 @@ let defaultFilter = {
   field: 'search',
   value: ''
 };
+
 class Filter extends Component {
   state = {
-    ...defaultFilter
+    ...defaultFilter,
+    filtered: false
   };
 
-  apply_filter = React.createRef();
-
   handleInputChange = event => {
     const target = event.target;
     const value = target.type === 'checkbox' ? target.checked : target.value;
     const name = target.name;
 
     this.setState({
-      [name]: value
+      [name]: value,
+      filtered: false
     });
   };
 
   onSubmit = event => {
     event.preventDefault();
     const filterString = '&' + this.state.field + '=' + this.state.value;
     this.props.applyFilter(filterString);
+    this.setState({
+      filtered: true
+    });
   };
 
   clearFilter = () => {
     this.props.clearFilter();
-    this.apply_filter.classList.remove('filter__apply-btn-active');
     this.setState({
-      ...defaultFilter
+      ...defaultFilter,
+      filtered: false
     });
   };
 
   renderFields = () => {
-    const { columns } = this.props;
-    
+    const {columns} = this.props;
+
     let fields = columns.map(column => (
       <option key={column.field} value={column.field}>{column.title}</option>
     ));
 
-    fields.unshift(<option key="search" value="search">Все</option>);
-      
+    fields.unshift(<option key='search' value='search'>Все</option>);
```

Comment:
Double quotes are used in JSX attributes by convention.

Reply:
where I can read this convention ?